datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/licenses.py

```diff
@@ -35,8 +35,6 @@
     'aerospike': ['Apache-2.0'],
     # https://github.com/baztian/jaydebeapi/blob/master/COPYING
     'JayDeBeApi': ['LGPL-3.0-only'],
-    # https://github.com/mhammond/pywin32/blob/master/adodbapi/license.txt
-    'adodbapi': ['LGPL-2.1-only'],
```

Comment:
To reviewer: this dependency was removed [here](https://github.com/DataDog/integrations-core/pull/13843)